SMLHelper/mod.json

```diff
@@ -4,7 +4,7 @@
   "Author": "AHK1221 & PrimeSonic & AlexejheroYTB",
   "Version": "2.3.0",
   "Enable": true,
-  "Game": "Subnautica",
+  "Game": "BelowZero",
```

Comment:
**TODO**
Create separate mod.json files for each game.  
Update version to `2.3.1`.

Reply:
After separating this into two files, a simple post build event was added that copies the appropriate file into the output folder with the name `mod.json`